core/services/privacy_policy/Manager.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace EventEspresso\core\services\privacy_policy;
+
+use EventEspresso\core\exceptions\InvalidClassException;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidEntityException;
+use EventEspresso\core\exceptions\InvalidFilePathException;
+use EventEspresso\core\exceptions\InvalidIdentifierException;
+use EventEspresso\core\exceptions\InvalidInterfaceException;
+use EventEspresso\core\services\collections\CollectionDetails;
+use EventEspresso\core\services\collections\CollectionInterface;
+use EventEspresso\core\services\collections\CollectionLoader;
+use WP_Post;
+
+defined('EVENT_ESPRESSO_VERSION') || exit('No direct script access allowed');
+
+
+
+/**
+ * Class PrivacyPolicyManager
+ * Manages setting up the hooks to add the EE core and add-ons' privacy policies
+ *
+ * @package        Event Espresso
+ * @author         Mike Nelson
+ * @since          $VID:$
+ */
+class Manager
+{
+
+    public function __construct()
+    {
+        add_action( 'edit_form_after_title', array( $this, 'possiblyAddPrivacyPolicy' ), 9 );
+    }
+
+
+
+    /**
+     * For all the registered `PrivacyPolicyGeneratorInterface`s, add their privacy policy content
+     * @param WP_Post $post
+     */
+    public function possiblyAddPrivacyPolicy($post)
```

Comment:
another nitpicky issue... can we plz just call this `addPrivacyPolicy()` ?
I understand that it is not guaranteed to always add the privacy policy content to the supplied post, but the existence of some conditionals to bypass that process should not warrant prepending the method name with "possibly". If that were the case, then a huge number of methods within our codebase, especially any with conditionals that result in an early exit, could be similarly prefixed with "possibly", but that would be silly to see everywhere wouldn't it?

Reply:
hmmm ya, I suppose there are a lot of functions that don't necessarily perform their described action based on some conditions. But I think those usually don't perform their action because of some error, whereas this method expects to USUALLY not perform this action because usually users aren't on this specific page.
I'd like to just go along with your suggestion, but it gets less clear if I, someday, decide to put all the policy-adding code into a new function (which ideally would also be named `addPrivacyPolicy`), and call it from this one (which you're suggesting should have that same name).
Would the prefix "maybe" work better? (I see it's been used a bit.) I'd be inclined to only add it when it's anticipated that the function won't perform its action in non-error conditions.